index.html

```diff
@@ -76,6 +76,26 @@
     <li>Kopi Po Siu Dai</li>
   </ul>
 </div>
+<div class='wide-container' id='make-own-kopi'>
```

Comment:
Use double quotes instead of single quotes here. E.g. `class="wide-container"` 😁